src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -1092,6 +1092,47 @@ private Autogenerated.StateItem ToAutogeneratedStateItem(StateTransactionRequest
         }
         #endregion
 
+        #region Configuration API
+        /// <inheritdoc/>
+        public override Task<IReadOnlyList<ConfigurationItem>> GetConfiguration(
+            string storeName,
+            IReadOnlyList<string> keys,
+            IReadOnlyDictionary<string, string> metadata = default,
+            CancellationToken token = default)
+        {
+            var request = new Autogenerated.GetConfigurationRequest()
```

Comment:
Add null check for `storeName`, would be good to tolerate `keys` being null since we said it's optional.

Reply:
Done.